components/alerts/alerts.css

```diff
@@ -0,0 +1,52 @@
+.alert {
+  font-size: 0.875rem;
+  padding: 1.5rem;
+  margin-bottom: 1rem;
+  border: none;
+  border-radius: 0.5rem;
+  transition: 0.2s ease-in-out;
+  display: flex;
```

Comment:
you have already defined a utility class for flex. Why not use that?

Reply:
Yes. Will fix that. Thanks.